packages/cli/src/project/project.ts

```diff
@@ -43,11 +43,20 @@ export class Project implements IProject {
   }
 
   static detectLanguage(): ProjectLanguage | undefined {
-    return fs.existsSync(path.join(process.cwd(), 'package.json'))
-      ? 'typescript'
-        : fs.readdirSync(process.cwd()).some(file => file.endsWith('.sln'))
-      ? 'csharp'
-        : undefined;
+    if (fs.existsSync(path.join(process.cwd(), 'package.json'))) {
+      return 'typescript';
+    }
+    if (fs.readdirSync(process.cwd()).some(file => file.endsWith('.sln'))) {
+      return 'csharp';
+    }
+    if (
+      fs.existsSync(path.join(process.cwd(), 'pyproject.toml')) &&
+      (process.env.ENABLE_EXPERIMENTAL_PYTHON_OPTIONS === 'true' ||
```

Comment:
thoughts on adding a note somewhere that python is on alpha / private preview, hence we have this experimental feat flag in place?